plugin_xiplot_filetypes/xiplot_filetypes/__init__.py

```diff
@@ -0,0 +1,44 @@
+from io import BytesIO
+
+import pandas as pd
+
+
+def read_parquet():
+    try:
```

Comment:
Why do we expect the import to fail, even when the plugin is installed?

Reply:
Yeah, the parquet one should never fail. Mostly a leftover from copy pasting (and maybe as a pattern for future file types).